README.md

```diff
@@ -67,6 +67,43 @@ configuration file located in `/etc/routinator/routinator.conf`. You can check
 the status of Routinator with `sudo systemctl status routinator` and view the
 logs with `sudo journalctl --unit=routinator`.
 
+## Quick Start with RHEL/CentOS Packages
+
+Assuming you have a machine running a RHEL/CentOS 7 or 8 distribution, you can 
+install Routinator, you can install Routinator from our [software package
```

Comment:
"you can install Routinator, you can install Routinator" - no need to say it twice ;-)